### PR TITLE
Convert NetworkUtils.java into kotlin

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes: #3021
Parent Issue: #1826 

Converted NetworkUtils.java into kotlin

### CODE REVIEW

File: core/src/main/java/org/kiwix/kiwixmobile/core/utils/NetworkUtils.kt [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import java.util.UUID
+
+object NetworkUtils {
+  /**
+   * check availability of any network
+   *
+   * @return true if a network is ready to be used
+   */
+  fun isNetworkAvailable(context: Context): Boolean {
+    val connectivity: ConnectivityManager? = context
```
Comment (gouri-panda) [inline core/src/main/java/org/kiwix/kiwixmobile/core/utils/NetworkUtils.kt:36]: It is a non-nullable type. So we can remove null.

Comment (AadeshDhimanDeveloper) [reply core/src/main/java/org/kiwix/kiwixmobile/core/utils/NetworkUtils.kt]: hi @gouri-panda  done

Comment (MohitMaliFtechiz) [reply core/src/main/java/org/kiwix/kiwixmobile/core/utils/NetworkUtils.kt]: hi @gouri-panda  done

**Comment by kelson42** [comment]
@MohitMaliDeveloper Please as to re-review if this is reado to re-review. By not doing so, we just loose time for no reason.

**Review by gouri-panda — APPROVED** [review]

**Pull request merged**